Description: Install GeoDataTreeModel.h
Origin: upstream, https://projects.kde.org/projects/kde/kdeedu/marble/repository/revisions/c644c3cbcc
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1049473

--- a/src/lib/CMakeLists.txt
+++ b/src/lib/CMakeLists.txt
@@ -346,6 +346,7 @@
     SunLocator.h
     ClipPainter.h
     GeoGraphicsScene.h
+    GeoDataTreeModel.h
     geodata/data/GeoDataAbstractView.h
     geodata/data/GeoDataAccuracy.h
     geodata/data/GeoDataColorStyle.h
